src/lib/mira_integration/mira_integration_timer.c

```diff
@@ -67,9 +74,55 @@ void vPortSetupTimerInterrupt(
     current_tick_time = previous_tick_time;
     /* Trigger SysTick exception */
     SCB->ICSR = SCB_ICSR_PENDSTSET_Msk;
+
+    /* Wakeup routine depends on WFE waking up from IRQ */
+    /* also needed for ERRATA 220 fix for 52832 */
+    SCB->SCR |= SCB_SCR_SEVONPEND_Msk;
 }
 
 #if (configUSE_TICKLESS_IDLE == 1)
+
+/**
+ * Wrapper for WFE instruction, with fixes for nordic erratas
+ *
+ * To be called with interrupts disabled
+ */
+static void softdevice_wait_for_event(
```

Comment:
Why is this prefixed with softdevice? Seems generic to me.

Reply:
It is actually generic.

Reply:
Sounds like something that definitely can be changed. Think it's simply because it came from `cpu_softdevice_wait_for_event`, internal function in mira, then got updated a couple of times. So a better name makes sense.

Reply:
Done!